src/main/java/at/pfeifer/chatapp/controller/ChatController.java

```diff
@@ -30,14 +30,21 @@ public class ChatController implements Initializable {
     @FXML
     private Button sendButton;
 
+    @FXML
+    private TextField userName;
+
+    String userNameText;
+
     @FXML
     void sendMessage() {
         String message = messageInput.getText();
+
         if (message.isEmpty()) return;
+        String messageWithUserName = combineUserNameWithMessage(message, userNameText);
```

Comment:
```suggestion
        String messageWithSignature= combineUserNameWithMessage(message, signatureInput.getText());
        
```
Just call the getText function. Storing it in a temporary String is uneccessary in this case